CONTRIBUTING.md

```diff
@@ -89,150 +89,175 @@ effect on highlighting. We will work on improving highlighting in the near futur
-@constant.builtin
-@constant.macro
-@string
-@string.regex
-@string.escape
-@string.special
-@character
-@character.special
-@number
-@boolean
-@float
+
+#### Literals
+
+```scheme
+@string            ; string literals
+@string.regex      ; regular expressions
+@string.escape     ; escape sequences
+@string.special    ; other special strings (e.g. dates)
+
+@character         ; character literals
+@character.special ; special characters (e.g. wildcards)
+
+@boolean           ; boolean literals
+@number            ; numeric literals
+@float             ; floating-point number literals
 ```
 
 #### Functions
 
-```
-@function
-@function.builtin
-@function.call
-@function.macro
-@parameter
-
-@method
-@method.call
-@field
-@property
-
-@constructor
+```scheme
+@function         ; function definitions
+@function.builtin ; built-in functions
+@function.call    ; function calls
+@function.macro   ; preprocessor macros
+
+@method           ; method definitions
+@method.call      ; method calls
+
+@constructor      ; constructor calls and definitions
+@parameter        ; parameters of a function
 ```
 
 #### Keywords
 
-```
-@conditional (e.g. `if`, `else`)
-@repeat (e.g. `for`, `while`)
-@label for C/Lua-like labels
-@keyword
-@keyword.function (keyword to define a function, e.g. `func` in Go, `def` in Python)
-@keyword.operator (for operators that are English words, e.g. `and`, `or`)
-@keyword.return
-@operator (for symbolic operators, e.g. `+`, `*`)
-@exception (e.g. `throw`, `catch`)
-@include keywords for including modules (e.g. import/from in Python)
-@storageclass
-
-@type
-@type.builtin
-@type.definition
-@type.qualifier
-@namespace for identifiers referring to namespaces
-@symbol for identifiers referring to symbols
-@attribute for e.g. Python decorators
+```scheme
+@keyword          ; various keywords
+@keyword.function ; keywords that define a function (e.g. `func` in Go, `def` in Python)
+@keyword.operator ; operators that are English words (e.g. `and` / `or`)
+@keyword.return   ; keywords like `return` and `yield`
+
+@conditional      ; keywords related to conditionals (e.g. `if` / `else`)
+@repeat           ; keywords related to loops (e.g. `for` / `while`)
+@debug            ; keywords related to debugging
+@label            ; GOTO and other labels (e.g. `label:` in C)
+@include          ; keywords for including modules (e.g. `import` / `from` in Python)
+@exception        ; keywords related to exceptions (e.g. `throw` / `catch`)
 ```
 
-@conceal followed by `(#set! conceal "")` for captures that are not used for highlights but only for concealing.
+#### Types
 
-#### Variables
+```scheme
+@type            ; type or class definitions and annotations
+@type.builtin    ; built-in types
+@type.definition ; type definitions (e.g. `typedef` in C)
+@type.qualifier  ; type qualifiers (e.g. `const`)
 
+@storageclass    ; visibility/life-time/etc. modifiers (e.g. `static`)
+@attribute       ; attribute annotations (e.g. Python decorators)
```

Comment:
> I plan to submit another PR soon that will make some queries more compliant with the documentation and use most if not all of the unused captures. I can drop the note but I'd rather not remove them entirely for the time being.

I imagine this is the reason why this is not the case just yet for Python decorators :hugs: